common/views/themes/grid/index.ts

```diff
@@ -1,6 +1,3 @@
 import { gridBase } from './grid-base';
-import { gridScroll } from './grid-scroll';
-import { gridSize } from './grid-size';
-import { gridTheme } from './grid-theme';
 
-export const grid = [gridBase, gridScroll, gridSize, gridTheme].join(' ');
+export const grid = [gridBase].join(' ');
```

Comment:
Isn't this just equivalent to `gridBase` now?

Reply:
It sure is! Will update